src/fsharp/CompileOps.fs

```diff
@@ -826,10 +826,16 @@ let OutputPhasedErrorR (os: StringBuilder) (err: PhasedDiagnostic) (suggestNames
       | UndefinedName(_, k, id, suggestionsF) ->
           os.Append(k (DecompileOpName id.idText)) |> ignore
           if suggestNames then
-              let filtered = ErrorResolutionHints.FilterPredictions suggestionsF id.idText 
-              if List.isEmpty filtered |> not then
-                  os.Append(ErrorResolutionHints.FormatPredictions DecompileOpName filtered) |> ignore
-          
+              let buffer = ErrorResolutionHints.SuggestionBuffer(id.idText)
+              suggestionsF buffer.Add
+              let values = buffer.Values()
+              if not (Array.isEmpty values) then
+                  os.Append " " |> ignore
```

Comment:
Why add a blank string here?

Reply:
it's to separate both texts and it's only one time thing